gunicorn/util.py

```diff
@@ -232,33 +232,29 @@ def is_ipv6(addr):
     return True
 
 
-def parse_address(netloc, default_port=8000):
+def parse_address(netloc, default_port='8000'):
+    host, port = [netloc, default_port]
```

Comment:
I would move this down to an `else` statement or at least after these first two blocks. Probably an `else` because I like branching where all assignments are at the same level/depth. ;) That would also make sure the `tcp://` stripping take effect. Right now, `tcp://` with an address but no port might not work.